Source/MQTTnet.Extensions.ManagedClient/ManagedMqttClient.cs

```diff
@@ -124,6 +125,12 @@ public ManagedMqttClient(IMqttClient mqttClient, IMqttNetLogger logger)
             remove => _synchronizingSubscriptionsFailedEvent.RemoveHandler(value);
         }
 
+        public event Func<SubscriptionsResultEventArgs, Task> SubscriptionsResultAsync
```

Comment:
I recommend renaming this to something like "SubscriptionsChangedAsync" because it indicates WHEN the event is fired.